### PR TITLE
Editorial changes

### PR DESCRIPTION
**Pull request by AFBarstow**
Nice document! Some minor editorial changes to consider.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -327,7 +327,7 @@
 
           <section>
         <h3>Browser Support</h3>
-        <p><a href="#fig7">Fig.7</a> is the support data table for [[NAVIGATION-TIMING]], generated by <a href="http://caniuse.com/#feat=nav-timing">caniuse.com</a>. If you are cursious about the status of the latest Navigation Timing design in Level 2, the Web Performance WG has provided a <a href="http://www.w3.org/2015/10/webperf-all.html">detail report</a>(by October 21, 2015) for the latest features in its deliverables.</p>
+        <p><a href="#fig7">Fig.7</a> is the support data table for [[NAVIGATION-TIMING]], generated by <a href="http://caniuse.com/#feat=nav-timing">caniuse.com</a>. If you are cursious about the status of the latest Navigation Timing design in Level 2, the Web Performance WG has provided a <a href="http://www.w3.org/2015/10/webperf-all.html">details report </a>(by October 21, 2015) for the latest features in its deliverables.</p>
```
Comment (yoavweiss) [inline index.html:330]: maybe "detailed"? Also, spurious space

File: index.html [hunk 2]
```diff
@@ -228,7 +228,7 @@
 
         <section>
           <h2>Time Origin</h2>
-          <p>Time Origin determines the time value from which time is measured. For a dedicated worker or a document the origin time is the start of page navigation of that document, and for a shared worker the origin time is when the shared worker is created. For a more accurate defination, please check the spec <a href="http://www.w3.org/TR/hr-time-2/#time-origin-1">detail</a>.</p>
+          <p>Time Origin determines the time value from which time is measured. For a dedicated worker or a document the origin time is the start of page navigation of that document, and for a shared worker the origin time is when the shared worker is created. For a more accurate defination, please check the spec's <a href="http://www.w3.org/TR/hr-time-2/#time-origin-1">detail</a>.</p>
```
Comment (yoavweiss) [inline index.html:231]: maybe "check the spec for details"?

**Comment by yoavweiss** [comment]
LGTM % nits

**Comment by AFBarstow** [comment]
Your 2nd suggestion would be fine but note the doc has other spelling errors including "cursious" on the line we are discussing ;-). 

Perhaps you could merge my PR then I'll submit a new one that tweaks the "details" line and includes fixes for other editorial changes.

**Comment by yoavweiss** [comment]
WFM :) Merging

**Pull request merged**